app/redirect.rb

```diff
@@ -44,6 +44,10 @@ def checkRedirect(origURI)
            "pub-jschol-prd.escholarship.org"
           ].include?(uri.host)
       uri.host = "escholarship.org"
+    elsif uri.host == "repositories.cdlib.org"
```

Comment:
In a conversation with @martinhaye earlier today, I think he was suggesting that the order here is important, and we may want to find another place for these so that your new tests pass.

Reply:
obviously the order is important.  but, judging from the failure, I think the problem in now in `handleBepressRedirect` (which probably needs a test too) and `handleDojRedirect`, is it not, because as far as I see that is the only place that would send a redirect to https://escholarship.org/uc/doj

With the previous order, the handle...Redriect's were not even getting called for this URL

Reply:
I've given this a couple hours today, trying to rewrite the logic in the checkRedirect method so that the new test will pass, in order to move this along. I'm afraid I can't really trace the existing logic at all, so, I'm throwing in the towel.  In the role of reviewer of a PR, I can only say: this suggested change does not pass the provided test.